src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -558,18 +558,26 @@ export class TerminalService extends Disposable implements ITerminalService {
 		return instance;
 	}
 
-	async revealActiveTerminal(preserveFocus?: boolean): Promise<void> {
-		const instance = this.activeInstance;
-		if (!instance) {
+	async revealTerminal(source: ITerminalInstance, preserveFocus?: boolean): Promise<void> {
+		if (!source) {
```

Comment:
is it possible that `source` is undefined?

I think we can remove this check?

Reply:
You're right.